src/Random.h

```diff
@@ -30,12 +30,7 @@
 // Description: Declarations for a class dealing with random numbers.
 ///////////////////////////////////////////////////////////////////////////////
 
-#ifdef USE_BOOST_RANDOM
```

Comment:
should use `std::random`

Reply:
what do you mean?

Reply:
Oh right yes makes sense. I can replace so multineat always works with random objects instead of the global state.